.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    name: Run unit tests
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run tests inside docker container
+        uses: addnab/docker-run-action@v3
```

Comment:
Forked to https://github.com/manticoresoftware/addnab-docker-run-action . Pls use it from there